docs/pages/release_notes.rst

```diff
@@ -5,6 +5,18 @@ Release notes
 
 What has been changed, and how to update between versions.
 
+*****************
+Hazelnut (to be released) (v6.0.0)
```

Comment:
Current master branch is elected to become v5.1. This will create a conflict.

Reply:
Will wait with merging after v5.1 got released